backend/src/main/java/ai/giskard/web/rest/controllers/InspectionController.java

```diff
@@ -74,6 +78,32 @@ public JsonNode getRowsFiltered(@PathVariable @NotNull Long inspectionId, @Reque
         return result;
     }
 
+    /**
+     * This function asks for a slice ID and an inspection ID and returns the list of rows filtered thru that slice.
+     * It streams data to the ML worker and is a bit more complex in its execution
+     * @param inspectionId
+     * @param sliceId
+     * @return
+     */
+    @PostMapping("/inspection/{inspectionId}/slice/{sliceId}")
+    @Transactional
+    public JsonNode getRowsSliced(@PathVariable @NotNull Long inspectionId, @PathVariable @NotNull Long sliceId) throws IOException {
```

Comment:
Instead of adding a new method I think we should add sliceId to the existing `getRowsFiltered` otherwise slicing + filtering doesn't work